test/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_Native_v1.t.sol

```diff
@@ -0,0 +1,355 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {Clones} from "@oz/proxy/Clones.sol";
+import {BancorFormula} from
+    "src/modules/fundingManager/bondingCurve/formulas/BancorFormula.sol";
+import {
+    IFM_BC_Bancor_Redeeming_VirtualSupply_v1,
+    FM_BC_Bancor_Redeeming_VirtualSupply_v1,
+    IFundingManager_v1
+} from
+    "src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol";
+
+import {
+    ModuleTest,
+    IModule_v1,
+    IOrchestrator_v1
+} from "test/modules/ModuleTest.sol";
+import {OZErrors} from "test/utils/errors/OZErrors.sol";
+
+import {IFM_BC_Bancor_Redeeming_VirtualSupply_Native_v1} from
+    "@fm/bondingCurve/interfaces/IFM_BC_Bancor_Redeeming_VirtualSupply_Native_v1.sol";
+import {NativeIssuance_v1} from "@ex/token/NativeIssuance_v1.sol";
+import {FM_BC_Bancor_Redeeming_VirtualSupply_Native_v1} from
+    "@fm/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_Native_v1.sol";
+import {FM_BC_Bancor_Redeeming_VirtualSupply_NativeV1Mock} from
+    "./utils/mocks/FM_BC_Bancor_Redeeming_VirtualSupply_NativeV1Mock.sol";
+import {NativeMinterMock} from "test/utils/mocks/external/NativeMinterMock.sol";
+
+contract FM_BC_Bancor_Redeeming_VirtualSupply_NativeV1Test is ModuleTest {
```

Comment:
Sam as with the native minter, BTT would be great :)

Reply:
added some comments on tests about what they really tests